Update UK.en.home.sitemap.json

TescoClick opened the pull request:
Updated Tesco.com URLs

dobbym commented:
Could you stick to the same formatting so the diff makes sense please?

TescoClick commented:
> Could you stick to the original indenting pattern please?

Resolved indenting issue and committed to Click Branch. Was this the only formatting issue?

dobbym commented on UK.en.home.sitemap.json:
```diff
@@ -1,151 +1,200 @@
+  { "url": "phones/products/05025813012380" },
+  { "url": "zones/pay-plus" },
+  { "url": "phones/products/05025813012366" },
+  { "url": "help/returns-policy/" },
```
Looks like this has a duplicate on #95

dobbym commented on UK.en.home.sitemap.json:
```diff
@@ -1,151 +1,200 @@
+  { "url": "phones/products/05025813009205" },
+  { "url": "phones/products/04894461781211" },
+  { "url": "phones/products/05000462191259" },
+  { "url": "help/terms-and-conditions/grocery/" },
```
Duplicate below

dobbym commented:
Should be live now https://www.tesco.com/UK.en.home.sitemap.xml

TescoClick commented:
Hi,

Just to let you know I have updated the sitemap in GitHub and sent through
the pull request.

Can you please confirm.

Thanks,

Stefan

On Thu, 26 Sep 2019 at 12:08, David Merrilees <notifications@github.com>
wrote:

> Should be live now https://www.tesco.com/UK.en.home.sitemap.xml
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dobbym/satsuma/pull/1?email_source=notifications&email_token=AM4YZDTFKBLOFHS2B4IMTNDQLSJZBA5CNFSM4I2OQ5V2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD7VGHOI#issuecomment-535454649>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AM4YZDR5AFJEOHFGZENYAYLQLSJZBANCNFSM4I2OQ5VQ>
> .
>


--

-- 


Click Consult Limited, Registration Number 4798223*
**
Please consider 
the environment before printing this e-mail*

*DISCLAIMER
*This
email and 
any files transmitted with it are confidential and intended solely
for the 
attention and use of the individual or entity to whom they are
addressed. 
No copyright or other intellectual rights to any material attached
to this 
email, either inline or as an attachment are transferred to the
recipient 
unless explicitly stated. If you have received this email in error
please 
reply to inform us accordingly, prior to deleting the message. Do not use, 
copy or disclose the information contained in this message or in any 
attachment.

dobbym commented:
Hi

I can't see any open pull requests. Did you push the branch, and open the
pull request?

On Mon, 6 Jan 2020 at 11:58, TescoClick <notifications@github.com> wrote:

> Hi,
>
> Just to let you know I have updated the sitemap in GitHub and sent through
> the pull request.
>
> Can you please confirm.
>
> Thanks,
>
> Stefan
>
> On Thu, 26 Sep 2019 at 12:08, David Merrilees <notifications@github.com>
> wrote:
>
> > Should be live now https://www.tesco.com/UK.en.home.sitemap.xml
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/dobbym/satsuma/pull/1?email_source=notifications&email_token=AM4YZDTFKBLOFHS2B4IMTNDQLSJZBA5CNFSM4I2OQ5V2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD7VGHOI#issuecomment-535454649
> >,
> > or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AM4YZDR5AFJEOHFGZENYAYLQLSJZBANCNFSM4I2OQ5VQ
> >
> > .
> >
>
>
> --
>
> --
>
>
> Click Consult Limited, Registration Number 4798223*
> **
> Please consider
> the environment before printing this e-mail*
>
> *DISCLAIMER
> *This
> email and
> any files transmitted with it are confidential and intended solely
> for the
> attention and use of the individual or entity to whom they are
> addressed.
> No copyright or other intellectual rights to any material attached
> to this
> email, either inline or as an attachment are transferred to the
> recipient
> unless explicitly stated. If you have received this email in error
> please
> reply to inform us accordingly, prior to deleting the message. Do not use,
> copy or disclose the information contained in this message or in any
> attachment.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/dobbym/satsuma/pull/1?email_source=notifications&email_token=AAGLLO2PMBRVEF3C5H3EZ23Q4MMHDA5CNFSM4I2OQ5V2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIFH6DY#issuecomment-571113231>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGLLO42UF6H4FFHEZIQH2LQ4MMHDANCNFSM4I2OQ5VQ>
> .
>

dobbym commented:
Ah @TescoClick, you've pushed to an old branch. i'll copy the commit to a new branch and create a new PR.

Please create separate branches for each change in future.

TescoClick commented:
Yes commited to master branch.

Thanks for sorting this out. Will create a new branch in future. In other
github accounts I have, they usually prefer me to commit to a current
branch so will make a note for next time.

Thanks,

Stefan

On Mon, 6 Jan 2020 at 12:41, David Merrilees <notifications@github.com>
wrote:

> Ah @TescoClick <https://github.com/TescoClick>, you've pushed to an old
> branch. i'll copy the commit to a new branch and create a new PR.
>
> Please create separate branches for each change in future.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dobbym/satsuma/pull/1?email_source=notifications&email_token=AM4YZDQYRKHLWHU5DY72SGDQ4MRHNA5CNFSM4I2OQ5V2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIFKUJY#issuecomment-571124263>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AM4YZDX5BU7RRTUAJI5SNMTQ4MRHNANCNFSM4I2OQ5VQ>
> .
>


--

-- 


Click Consult Limited, Registration Number 4798223*
**
Please consider 
the environment before printing this e-mail*

*DISCLAIMER
*This
email and 
any files transmitted with it are confidential and intended solely
for the 
attention and use of the individual or entity to whom they are
addressed. 
No copyright or other intellectual rights to any material attached
to this 
email, either inline or as an attachment are transferred to the
recipient 
unless explicitly stated. If you have received this email in error
please 
reply to inform us accordingly, prior to deleting the message. Do not use, 
copy or disclose the information contained in this message or in any 
attachment.

dobbym commented:
Ok, as it's so small and if it's easier to you, we can work on 1 branch.

On Mon, 6 Jan 2020 at 12:44, TescoClick <notifications@github.com> wrote:

> Yes commited to master branch.
>
> Thanks for sorting this out. Will create a new branch in future. In other
> github accounts I have, they usually prefer me to commit to a current
> branch so will make a note for next time.
>
> Thanks,
>
> Stefan
>
> On Mon, 6 Jan 2020 at 12:41, David Merrilees <notifications@github.com>
> wrote:
>
> > Ah @TescoClick <https://github.com/TescoClick>, you've pushed to an old
> > branch. i'll copy the commit to a new branch and create a new PR.
> >
> > Please create separate branches for each change in future.
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/dobbym/satsuma/pull/1?email_source=notifications&email_token=AM4YZDQYRKHLWHU5DY72SGDQ4MRHNA5CNFSM4I2OQ5V2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIFKUJY#issuecomment-571124263
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AM4YZDX5BU7RRTUAJI5SNMTQ4MRHNANCNFSM4I2OQ5VQ
> >
> > .
> >
>
>
> --
>
> --
>
>
> Click Consult Limited, Registration Number 4798223*
> **
> Please consider
> the environment before printing this e-mail*
>
> *DISCLAIMER
> *This
> email and
> any files transmitted with it are confidential and intended solely
> for the
> attention and use of the individual or entity to whom they are
> addressed.
> No copyright or other intellectual rights to any material attached
> to this
> email, either inline or as an attachment are transferred to the
> recipient
> unless explicitly stated. If you have received this email in error
> please
> reply to inform us accordingly, prior to deleting the message. Do not use,
> copy or disclose the information contained in this message or in any
> attachment.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/dobbym/satsuma/pull/1?email_source=notifications&email_token=AAGLLO64BNGI7TUCGXALH5DQ4MRS7A5CNFSM4I2OQ5V2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIFK2PY#issuecomment-571125055>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGLLO64DB2U6G2UAKIIPADQ4MRS7ANCNFSM4I2OQ5VQ>
> .
>

dobbym commented:
Updated in production.

On Mon, 6 Jan 2020 at 13:10, David Merrilees <dobbym@googlemail.com> wrote:

> Ok, as it's so small and if it's easier to you, we can work on 1 branch.
>
> On Mon, 6 Jan 2020 at 12:44, TescoClick <notifications@github.com> wrote:
>
>> Yes commited to master branch.
>>
>> Thanks for sorting this out. Will create a new branch in future. In other
>> github accounts I have, they usually prefer me to commit to a current
>> branch so will make a note for next time.
>>
>> Thanks,
>>
>> Stefan
>>
>> On Mon, 6 Jan 2020 at 12:41, David Merrilees <notifications@github.com>
>> wrote:
>>
>> > Ah @TescoClick <https://github.com/TescoClick>, you've pushed to an old
>> > branch. i'll copy the commit to a new branch and create a new PR.
>> >
>> > Please create separate branches for each change in future.
>> >
>> > —
>> > You are receiving this because you were mentioned.
>> > Reply to this email directly, view it on GitHub
>> > <
>> https://github.com/dobbym/satsuma/pull/1?email_source=notifications&email_token=AM4YZDQYRKHLWHU5DY72SGDQ4MRHNA5CNFSM4I2OQ5V2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIFKUJY#issuecomment-571124263
>> >,
>> > or unsubscribe
>> > <
>> https://github.com/notifications/unsubscribe-auth/AM4YZDX5BU7RRTUAJI5SNMTQ4MRHNANCNFSM4I2OQ5VQ
>> >
>> > .
>> >
>>
>>
>> --
>>
>> --
>>
>>
>> Click Consult Limited, Registration Number 4798223*
>> **
>> Please consider
>> the environment before printing this e-mail*
>>
>> *DISCLAIMER
>> *This
>> email and
>> any files transmitted with it are confidential and intended solely
>> for the
>> attention and use of the individual or entity to whom they are
>> addressed.
>> No copyright or other intellectual rights to any material attached
>> to this
>> email, either inline or as an attachment are transferred to the
>> recipient
>> unless explicitly stated. If you have received this email in error
>> please
>> reply to inform us accordingly, prior to deleting the message. Do not
>> use,
>> copy or disclose the information contained in this message or in any
>> attachment.
>>
>> —
>> You are receiving this because you modified the open/close state.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/dobbym/satsuma/pull/1?email_source=notifications&email_token=AAGLLO64BNGI7TUCGXALH5DQ4MRS7A5CNFSM4I2OQ5V2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIFK2PY#issuecomment-571125055>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AAGLLO64DB2U6G2UAKIIPADQ4MRS7ANCNFSM4I2OQ5VQ>
>> .
>>
>

TescoClick commented:
Brilliant, thanks, Dave.


On Mon, 6 Jan 2020 at 15:33, David Merrilees <notifications@github.com>
wrote:

> Updated in production.
>
> On Mon, 6 Jan 2020 at 13:10, David Merrilees <dobbym@googlemail.com>
> wrote:
>
> > Ok, as it's so small and if it's easier to you, we can work on 1 branch.
> >
> > On Mon, 6 Jan 2020 at 12:44, TescoClick <notifications@github.com>
> wrote:
> >
> >> Yes commited to master branch.
> >>
> >> Thanks for sorting this out. Will create a new branch in future. In
> other
> >> github accounts I have, they usually prefer me to commit to a current
> >> branch so will make a note for next time.
> >>
> >> Thanks,
> >>
> >> Stefan
> >>
> >> On Mon, 6 Jan 2020 at 12:41, David Merrilees <notifications@github.com>
> >> wrote:
> >>
> >> > Ah @TescoClick <https://github.com/TescoClick>, you've pushed to an
> old
> >> > branch. i'll copy the commit to a new branch and create a new PR.
> >> >
> >> > Please create separate branches for each change in future.
> >> >
> >> > —
> >> > You are receiving this because you were mentioned.
> >> > Reply to this email directly, view it on GitHub
> >> > <
> >>
> https://github.com/dobbym/satsuma/pull/1?email_source=notifications&email_token=AM4YZDQYRKHLWHU5DY72SGDQ4MRHNA5CNFSM4I2OQ5V2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIFKUJY#issuecomment-571124263
> >> >,
> >> > or unsubscribe
> >> > <
> >>
> https://github.com/notifications/unsubscribe-auth/AM4YZDX5BU7RRTUAJI5SNMTQ4MRHNANCNFSM4I2OQ5VQ
> >> >
> >> > .
> >> >
> >>
> >>
> >> --
> >>
> >> --
> >>
> >>
> >> Click Consult Limited, Registration Number 4798223*
> >> **
> >> Please consider
> >> the environment before printing this e-mail*
> >>
> >> *DISCLAIMER
> >> *This
> >> email and
> >> any files transmitted with it are confidential and intended solely
> >> for the
> >> attention and use of the individual or entity to whom they are
> >> addressed.
> >> No copyright or other intellectual rights to any material attached
> >> to this
> >> email, either inline or as an attachment are transferred to the
> >> recipient
> >> unless explicitly stated. If you have received this email in error
> >> please
> >> reply to inform us accordingly, prior to deleting the message. Do not
> >> use,
> >> copy or disclose the information contained in this message or in any
> >> attachment.
> >>
> >> —
> >> You are receiving this because you modified the open/close state.
> >> Reply to this email directly, view it on GitHub
> >> <
> https://github.com/dobbym/satsuma/pull/1?email_source=notifications&email_token=AAGLLO64BNGI7TUCGXALH5DQ4MRS7A5CNFSM4I2OQ5V2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIFK2PY#issuecomment-571125055
> >,
> >> or unsubscribe
> >> <
> https://github.com/notifications/unsubscribe-auth/AAGLLO64DB2U6G2UAKIIPADQ4MRS7ANCNFSM4I2OQ5VQ
> >
> >> .
> >>
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dobbym/satsuma/pull/1?email_source=notifications&email_token=AM4YZDQQN7RL4EWCFCF2NW3Q4NFNRA5CNFSM4I2OQ5V2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIFZOKQ#issuecomment-571184938>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AM4YZDRB6FQAU4KVXG4TBQLQ4NFNRANCNFSM4I2OQ5VQ>
> .
>


--

-- 


Click Consult Limited, Registration Number 4798223*
**
Please consider 
the environment before printing this e-mail*

*DISCLAIMER
*This
email and 
any files transmitted with it are confidential and intended solely
for the 
attention and use of the individual or entity to whom they are
addressed. 
No copyright or other intellectual rights to any material attached
to this 
email, either inline or as an attachment are transferred to the
recipient 
unless explicitly stated. If you have received this email in error
please 
reply to inform us accordingly, prior to deleting the message. Do not use, 
copy or disclose the information contained in this message or in any 
attachment.
